app/src/main/java/com/eveningoutpost/dexdrip/Models/AlertType.java

```diff
@@ -118,14 +118,23 @@ public static AlertType get_highest_active_alert(Context context, double bg) {
 
         Boolean bg_unclear_readings_alerts = prefs.getBoolean("bg_unclear_readings_alerts", false);
         Long UnclearTimeSetting = Long.parseLong(prefs.getString("bg_unclear_readings_minutes", "90")) * 60000;
+        Boolean bg_unclear_readings_alerts_highvolume = prefs.getBoolean("bg_unclear_readings_alerts_highvolume", true);
 
         Long UnclearTime = BgReading.getUnclearTime(UnclearTimeSetting);
         AlertType at;
         if (UnclearTime >= UnclearTimeSetting && bg_unclear_readings_alerts ) {
             Log.w("NOTIFICATIONS", "Readings have been unclear for too long!!");
             Notifications.bgUnclearAlert(context);
+            if(bg_unclear_readings_alerts_highvolume) {
```

Comment:
Just wanted to make sure that I'll always be waken from sleep, in such a case. On a tablet, this alerts were not strong enough.
Please note that it has a setting of itself and is disabled by default.